### PR TITLE
chore(*): stackblitz conversion

### PR DESCRIPTION
**Pull request by amcdnl**
This PR converts plunker demos to stackblitz.

### CODE REVIEW

**Comment by amcdnl** [comment]
FYI - The current API for creating is `aio` ( https://github.com/angular/material.angular.io/pull/331/files#diff-f4d2d04dd15dd9caac319319cad1a1c4R7 ), @ericsimons is gonna change it to just `angular`.

**Comment by amcdnl** [comment]
This PR requires - https://github.com/angular/material2/pull/8546

File: src/app/shared/stackblitz/stackblitz-writer.ts [hunk 1]
```diff
@@ -2,50 +2,80 @@ import {Injectable} from '@angular/core';
...
  *   files[directory-name/file-name2]: file-content2,
  *   // Can add multiple tags
  *   tags[0]: tag-0,
- *   // Description of plunker
+ *   // Description of stackblitz
  *   description: description,
  *   // Private or not
  *   private: true
+ *  // Depedencies
+ *  depedencies: depedencies
```
Comment (willshowell) [inline src/app/shared/stackblitz/stackblitz-writer.ts]: typo: `dependencies`

File: src/app/shared/stackblitz/stackblitz-writer.ts [hunk 2]
```diff
@@ -2,50 +2,80 @@ import {Injectable} from '@angular/core';
...
+  'hammerjs': '^2.0.8'
+};
+
 /**
- * Plunker writer, write example files to Plunker
+ * Stackblitz writer, write example files to stackblitz
  *
- * Plunker API
+ * StackBlitz API
  * URL: http://plnkr.co/edit/?p=preview
```
Comment (willshowell) [inline src/app/shared/stackblitz/stackblitz-writer.ts]: Update url

File: src/app/shared/stackblitz/stackblitz-writer.ts [hunk 3]
```diff
@@ -2,50 +2,80 @@ import {Injectable} from '@angular/core';
...
+  'index.html',
+  'styles.scss',
+  'polyfills.ts',
+  '.angular-cli.json',
+  'main.ts'
+];
 
 const TAGS: string[] = ['angular', 'material', 'example'];
 
+const depedencies = {
```
Comment (willshowell) [inline src/app/shared/stackblitz/stackblitz-writer.ts]: typo: `dependencies`

**Comment by amcdnl** [comment]
@willshowell - Fixed; thanks!

File: src/app/shared/stackblitz/stackblitz-writer.ts [hunk 4]
```diff
@@ -2,50 +2,80 @@ import {Injectable} from '@angular/core';
...
+  'polyfills.ts',
+  '.angular-cli.json',
+  'main.ts'
+];
 
 const TAGS: string[] = ['angular', 'material', 'example'];
 
+const dependencies = {
+  '@angular/cdk': '^5.0.0-rc0',
+  '@angular/material': '^5.0.0-rc0',
```
Comment (jelbourn) [inline src/app/shared/stackblitz/stackblitz-writer.ts]: rc.1 now

File: src/app/shared/stackblitz/stackblitz-writer.ts [hunk 5]
```diff
@@ -2,50 +2,80 @@ import {Injectable} from '@angular/core';
...
+  'styles.scss',
+  'polyfills.ts',
+  '.angular-cli.json',
+  'main.ts'
+];
 
 const TAGS: string[] = ['angular', 'material', 'example'];
 
+const dependencies = {
+  '@angular/cdk': '^5.0.0-rc0',
```
Comment (jelbourn) [inline src/app/shared/stackblitz/stackblitz-writer.ts]: Break out the versions for the two release tracks?
```ts
const angularVersion = '^5.0.0';
const materialVersion = '^5.0.0-rc.1';
```

**Comment by amcdnl** [comment]
Done and bumped to new api

**Pull request merged**